lib/event_bg_bus.rb

```diff
@@ -21,10 +22,25 @@ class << self
     #
     def publish(event_name, payload = {})
       case event_name
-      when Symbol, String
-        EventWorker.perform_async(event_name, payload) if validEventBGBusEvent?(event_name)
-        EventBridge.put_event(event_name, payload) if validEventBridgeEvent?(event_name)
-        self
+      when Symbol
```

Comment:
I don't think we are typically using symbols when calling but I am not sure why are splitting the handling in these cases?

Reply:
Yeah I can't find any symbol usage so maybe we can just remove that case?